pkg/innerring/processors/audit/processor.go

```diff
@@ -35,12 +35,20 @@ type (
 		Reset() int
 	}
 
+	// EpochSource is an interface that provides actual
+	// epoch information.
+	EpochSource interface {
+		// EpochCounter must return current epoch number.
+		EpochCounter() uint64
+	}
```

Comment:
`pkg/core/netmap.State`?

Reply:
it has different method names, also we have that approach (a separate interface in the processor) in [netmap processor](https://github.com/nspcc-dev/neofs-node/blob/6358f4d74670033b2a43a72a22b899c8701ad3fd/pkg/innerring/processors/netmap/processor.go#L26) and some other processors

do you want me to add another wrapper in the innerring constructor or to unify all the netmap state interfaces and put it to the `pkg/core/netmap.State`?